### PR TITLE
🐈 Upgrade deps; clarify flow-typed command.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
platypus

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
@wlonk Can I get your 👍 here when you get a chance?

File: test/js/components/products/detail.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { MemoryRouter } from 'react-router-dom';
+import { StaticRouter } from 'react-router-dom';
```
Comment (wlonk) [inline test/js/components/products/detail.test.js]: Why this change to `StaticRouter`?

Comment (jgerigmeyer) [reply test/js/components/products/detail.test.js]: I don't know for sure -- some change to the internals of react-router.
